internal/core/metadata/rest_device.go

```diff
@@ -707,7 +735,17 @@ func restSetDeviceLastConnectedByName(
 
 	ctx := r.Context()
 	// Update last connected
-	if err = setLastConnected(d, lc, false, w, ctx, loggingClient, dbClient, errorHandler, nc); err != nil {
+	if err = setLastConnected(
+		d,
+		lc,
+		false,
+		w,
+		ctx,
+		loggingClient,
+		dbClient,
+		errorHandler,
+		nc,
+		config); err != nil {
```

Comment:
This all fits within the 120 character limit.

```
	if err = setLastConnected(d, lc, false, w, ctx, loggingClient, dbClient, errorHandler, nc, config); err != nil {
		return
	}
```
If for some reason it didn't, my preferred way to preserve readability would be to break the assignment out of the if; i.e. 

```
	err = setLastConnected(d, lc, false, w, ctx, loggingClient, dbClient, errorHandler, nc, config);
	if err != nil {
		return
	}
```

or 

```
	err = setLastConnected(
		d,
		lc,
		false,
		w,
		ctx,
		loggingClient,
		dbClient,
		errorHandler,
		nc,
		config);
	if err != nil {
		return
	}
```
Commented once; occurs elsewhere.

Reply:
Did not fit, hence the use of the second option suggested above 👍